ios/engine/KMEI/Demos/KeymanEngineDemo/AppDelegate.swift

```diff
@@ -12,34 +12,34 @@ import UIKit
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
   var window: UIWindow?
-
+  
```

Comment:
I'm pretty sure we trim empty lines, rather than preserve base indentation.

Reply:
> I'm pretty sure we trim empty lines, rather than preserve base indentation.

The Xcode Re-Indent command worked pretty well on the mac code, but mostly had the effect of adding spaces in the ios code. I think I'll trash this, and change only where it makes a difference.